src/structures/Message.js

```diff
@@ -155,6 +148,19 @@ class Message {
      * @private
      */
     this._edits = [];
+
+    if (data.member && this.guild && this.author) {
+      if (!this.guild.members.has(data.member.id)) {
```

Comment:
I think those ifs should be combined. :eyes:

Reply:
Resolved by dcabafb